src/PrestaShopBundle/Resources/views/Admin/Layout/default_layout.html.twig

```diff
@@ -0,0 +1,303 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+<!DOCTYPE html>
+<html lang="{{ iso_user }}">
+<head>
+  {% block header %}
+    {{ include('@PrestaShop/Admin/Layout/header.html.twig') }}
+    {% block stylesheets %}{% endblock %}
+    {% block extra_stylesheets %}{% endblock %}
+  {% endblock %}
+</head>
+
+{# Should be handled somewhere else and set as global variable, the twig function won't be useful anymore then #}
+{% set js_router_metadata = js_router_metadata() %}
+<body
+  class="lang-{{ iso_user }}{% if lang_is_rtl %} lang-rtl{% endif %} {{ controller_name|escape|lower }}{% if collapse_menu %} page-sidebar-closed{% endif %}{% if is_multishop|default(false) %} multishop-enabled{% endif %}{% if lite_display|default(false) %} light_display_layout{% endif %}{% if debug_mode|default(false) %} developer-mode{% endif %}"
+  {% if js_router_metadata is defined and js_router_metadata.base_url is defined %}data-base-url="{{ js_router_metadata.base_url }}"{% endif %}
+  {% if js_router_metadata is defined and js_router_metadata.token %}data-token="{{ js_router_metadata.token }}"{% endif %}
+>
+
+{# Layout header #}
+{% if display_header %}
+  {% block layout_header %}
+    <header id="header" class="d-print-none">
+      <nav id="header_infos" class="main-header">
+        <button class="btn btn-primary-reverse onclick btn-lg unbind ajax-spinner"></button>
+
+        {# Logo #}
+        <i class="material-icons js-mobile-menu">menu</i>
+        <a id="header_logo" class="logo float-left" href="{{ default_tab_link }}"></a>
+        <span id="shop_version">{{ ps_version }}</span>
+
+        <div class="component" id="quick-access-container">
+          {{ render(controller('PrestaShopBundle\\Controller\\Admin\\LegacyController::smartyComponentAction', {
```

Comment:
I'd use a twig function instead of rendering a controller for performance reasons in all components.

{{ smarty(template, variables) }}